src/main/java/com/actiontech/dble/services/manager/response/ShowBackend.java

```diff
@@ -213,13 +213,15 @@ private static RowDataPacket getRow(BackendConnection c, String charset) {
         row.add(c.getCharsetName().getClient().getBytes());
         row.add(c.getCharsetName().getCollation().getBytes());
         row.add(c.getCharsetName().getResults().getBytes());
-        row.add((c.getBackendService().getTxIsolation() + "").getBytes());
-        row.add((c.getBackendService().isAutocommit() + "").getBytes());
-        row.add(StringUtil.encode(c.getBackendService().getStringOfSysVariables(), charset));
-        row.add(StringUtil.encode(c.getBackendService().getStringOfUsrVariables(), charset));
-        row.add(StringUtil.encode(c.getBackendService().getXaStatus().toString(), charset));
+        row.add(c.isClosed() ? new byte[]{0} : (c.getBackendService().getTxIsolation() + "").getBytes());
```

Comment:
what's the difference between `new byte[]{0} ` and `null`

Reply:
> what's the difference between `new byte[]{0} ` and `null`

When the connection is closed, the relationship between connection and service is released